src/main/java/com/dumbdogdiner/stickyapi/bukkit/command/AsyncCommand.java

```diff
@@ -181,16 +181,16 @@ public void setTabCompleter(TabCompleter completer) {
      */
     @Override
     public java.util.@NotNull List<String> tabComplete(@NotNull CommandSender sender, @NotNull String alias,
-            String[] args) throws CommandException, IllegalArgumentException {
+            String @org.jetbrains.annotations.Nullable [] args) throws CommandException, IllegalArgumentException {
         if (args == null)
             throw new NullPointerException("arguments to tabComplete cannot be null");
 
-        List<String> completions = null;
+        @org.jetbrains.annotations.Nullable List<String> completions = null;
```

Comment:
I believe these annotations are for the KLS to help infer things properly - don't think we need them in internal Java code.